wgpu-types/src/lib.rs

```diff
@@ -4,13 +4,16 @@
 
 #[cfg(feature = "peek-poke")]
 use peek_poke::PeekPoke;
-#[cfg(feature = "serde")]
-use serde::{Deserialize, Serialize};
+#[cfg(feature = "replay")]
```

Comment:
I think it might be confusing for people looking to enable serialization to have to enable `trace` or `replay`. Most people would probably look for a `serde` or `serde1` feature because that's the common place to look.

For example, users may just want to serialize their types, but shouldn't necessarily need to understand the association between serialization and trace/replay (even if trace/replay happen to also require serialization).

Reply:
Right, I agree. Also, I think there is value to separating those even outside of the trace/replay scope: quite often the user knows exactly which side is needed, and deriving both affects compile times quite a bit.
Perhaps, this is something to address by documentation?